pkgs/top-level/release-python.nix

```diff
@@ -12,6 +12,11 @@
   nixpkgsArgs ? { config = {
     allowUnfree = false;
     inHydra = true;
+    # Exceptional unsafe packages that we still build and distribute,
+    # so users choosing to allow don't have to rebuild them every time.
+    permittedInsecurePackages = [
+      "olm-3.2.16" # see PR #347899
+    ];
```

Comment:
This file is only used by python-updates, so not needed in this context.

Reply:
Ok, dropped.